server/rpc_server.go

```diff
@@ -218,8 +204,16 @@ func (r *RPCServer) safelyExecuteHTTPRequest(w http.ResponseWriter, req *http.Re
 		}
 	}()
 
-	// hand the request off to gorilla
-	r.mux.ServeHTTP(w, req)
+	TimedAndCounted(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		if req.Body != nil {
+			defer func() {
+				if err := req.Body.Close(); err != nil {
```

Comment:
why bother the error check? I haven't really seen that anywhere else, everywhere it's just `defer req.body.Close`

Reply:
In cases where `server.ServeHTTP` is hit directly instead of via a real HTTP server, the body could be nil. This is here after running into that too many times.

Reply:
oh! the error check! I was thinking you were talking about the nil check.

The err check is here because we used to have `errcheck` in the build pipeline: https://github.com/NYTimes/gizmo/blob/master/Makefile#L35

I'd like to turn it back on at some point.

Reply:
👍